### PR TITLE
updated ECC keygen TVLA documentation

### PR DESCRIPTION
**Pull request by mojtaba-bisheh**
[solving issue 346](https://github.com/chipsalliance/caliptra-rtl/issues/346)

### CODE REVIEW

File: docs/CaliptraHardwareSpecification.md [hunk 1]
```diff
@@ -1299,17 +1299,27 @@ In practice, observing a t-value greater than a specific threshold (mainly 4.5)
 ![](./images/TVLA_threshold.png)
 
 
+##### KeyGen TVLA
+
+We detected a leakage using TVLA in the HMAC_DRBG algorithm during ECC key generation, based on 150,000 power traces. The leakage originated from a part of the SHA512 function (w_data) that was not fully protected by masking. Same leakage is expected for HMAC operations.
```
Comment (steph-morton) [inline docs/CaliptraHardwareSpecification.md:1304]: is `Same leakage` accurate or was this intended to be `Some leakage`? If `Same leakage` is accurate, then recommend update to `The same leakage`

File: docs/CaliptraHardwareSpecification.md [hunk 2]
```diff
@@ -1299,17 +1299,27 @@ In practice, observing a t-value greater than a specific threshold (mainly 4.5)
...
 
+##### KeyGen TVLA
+
+We detected a leakage using TVLA in the HMAC_DRBG algorithm during ECC key generation, based on 150,000 power traces. The leakage originated from a part of the SHA512 function (w_data) that was not fully protected by masking. Same leakage is expected for HMAC operations. 
+
+*Figure 40: seed/nonce-dependent leakage detection using TVLA for ECC keygen after 150,000 traces*
+
+![](./images/TVLA_keygen.png)
+
+The issue is very unlikely to occur in practice, even though it exists in TVLA results. Therefore, we will address it in the next release.
```
Comment (steph-morton) [inline docs/CaliptraHardwareSpecification.md:1310]: Recommend updating `The issue` to `This leakage`

Comment (mojtaba-bisheh) [reply docs/CaliptraHardwareSpecification.md:1310]: Done!

File: docs/CaliptraHardwareSpecification.md (hunk 1 above)
Comment (mojtaba-bisheh) [reply docs/CaliptraHardwareSpecification.md:1304]: Done!

**Review by steph-morton — APPROVED** [review]

**Review by calebofearth — DISMISSED** [review]

**Review by calebofearth — APPROVED** [review]

**Review by steph-morton — APPROVED** [review]

**Pull request merged**